scripts/setup.js

```diff
@@ -26,11 +33,14 @@ if (undefined === appConfig.NUM_CONFIRMATIONS) {
 if (argv.ropsten) {
   appConfig.API_URL = 'https://ropsten.api.kickback.events'
   appConfig.GIT_COMMIT = getGitCommit()
+  appConfig.LOGROCKET_TOKEN = '5gnafo/kickback-ropsten'
 } else if (argv.rinkeby) {
   appConfig.API_URL = 'https://rinkeby.api.kickback.events'
   appConfig.GIT_COMMIT = getGitCommit()
+  appConfig.LOGROCKET_TOKEN = '5gnafo/kickback-rinkeby'
 } else if (argv.live) {
   appConfig.MIXPANEL_ID = '11a2f7a59470cdb46cb611c5d22876f2'
+  appConfig.LOGROCKET_LIVE = '5gnafo/kickback-live'
```

Comment:
Like image uploading, can people spam the network if we expose token?

Reply:
Yeah, but we have this problem with our mixpanel id as well.

The difference is that we really want to control cloudinary data carefully, whereas in this case the data controllers are another company.

Reply:
ah, so we don't care if someone spam their network.